aws-transfer-workflow/src/main/java/software/amazon/transfer/workflow/Converter.java

```diff
@@ -84,6 +86,46 @@ static WorkflowStep toSdk(
                         .sourceFileLocation(workflowStep.getCopyStepDetails().getSourceFileLocation())
                         .build());
             }
+            if (WorkflowStepType.DECRYPT.toString().equals(type) && workflowStep.getDecryptStepDetails() != null) {
+                S3InputFileLocation s3InputFileLocation = null;
+                EfsFileLocation efsFileLocation = null;
```

Comment:
nit: `EfsInputFileLocation` to be consistent with the `S3InputFileLocation` naming